apps/emqx_bridge_s3tables/src/emqx_bridge_s3tables_action_schema.erl

```diff
@@ -97,13 +97,52 @@ fields(aggregation) ->
                 }
             )},
         {max_records,
-            hoconsc:mk(
+            mk(
                 pos_integer(),
                 #{
                     required => false,
                     default => 100_000,
                     desc => ?DESC("aggregation_max_records")
                 }
+            )},
+        {container,
+            mk(
+                emqx_schema:mkunion(
+                    type,
+                    #{
+                        <<"avro">> => ref(container_avro),
+                        <<"parquet">> => ref(container_parquet)
+                    },
+                    avro
+                ),
+                #{
+                    default => #{<<"type">> => <<"avro">>},
+                    desc => ?DESC("aggregation_container")
+                }
+            )}
+    ];
+fields(container_avro) ->
+    [{type, mk(avro, #{desc => ?DESC("container_type_avro")})}];
+fields(container_parquet) ->
+    [
+        {type, mk(parquet, #{desc => ?DESC("container_type_parquet")})},
+        {write_old_list_structure,
+            mk(boolean(), #{default => false, importance => ?IMPORTANCE_HIDDEN})},
+        {enable_dictionary, mk(boolean(), #{default => false, importance => ?IMPORTANCE_HIDDEN})},
+        {default_compression,
+            mk(
+                hoconsc:enum([none, zstd, snappy]),
+                #{default => zstd, importance => ?IMPORTANCE_HIDDEN}
+            )},
+        {data_page_header_version,
+            mk(
+                hoconsc:enum([1, 2]),
+                #{default => 2, importance => ?IMPORTANCE_HIDDEN}
+            )},
+        {max_row_group_bytes,
+            mk(
+                emqx_schema:bytesize(),
+                #{default => <<"1MB">>, desc => ?DESC("container_parquet_max_row_group_bytes")}
```

Comment:
Curious: is there any motivation for selecting this value as default?

Reply:
:checkered_flag: